identity-service/src/models/solanaNotification.js

```diff
@@ -40,8 +41,18 @@ module.exports = (sequelize, DataTypes) => {
     slot: {
       type: DataTypes.INTEGER,
       allowNull: false
+    },
+    metadata: {
```

Comment:
just curious what this metadata will be used for now?

Reply:
in this case, it's data in tipping that can't cleanly map the notificiation + notificationAction schema we have going on. we'll see how useful it is going forward